Package.swift

```diff
@@ -18,11 +18,19 @@ import PackageDescription
 import Foundation
 
 var packageDependencies: [Package.Dependency] = [
-  .package(url: "https://github.com/apple/swift-protobuf.git", .upToNextMinor(from: "1.3.1")),
-  .package(url: "https://github.com/kylef/Commander.git", .upToNextMinor(from: "0.8.0")),
-  .package(url: "https://github.com/apple/swift-nio-zlib-support.git", .upToNextMinor(from: "1.0.0")),
-  .package(url: "https://github.com/apple/swift-nio.git", .upToNextMinor(from: "1.12.0")),
-  .package(url: "https://github.com/apple/swift-nio-nghttp2-support.git", .upToNextMinor(from: "1.0.0")),
+  // Official SwiftProtobuf library, for [de]serializing data to send on the wire.
+  .package(url: "https://github.com/apple/swift-protobuf.git", .upToNextMajor(from: "1.3.1")),
+  
+  // Command line argument parser for our auxiliary command line tools.
+  .package(url: "https://github.com/kylef/Commander.git", .upToNextMajor(from: "0.8.0")),
```

Comment:
Should this be `upToNextMinor` since it's not a 1.0 yet?

Reply:
Agreed. Done.